docker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:8.2
+
+ARG POSTGREST_VERSION
+
+# Shorthands to make commands easier to read
+ENV POSTGREST_DOWNLOAD postgrest-${POSTGREST_VERSION}-ubuntu.tar.xz
+ENV POSTGREST_RELEASES_URL http://github.com/begriffs/postgrest/releases/download/v${POSTGREST_VERSION}
+
+RUN apt-get update \
+  && apt-get install -y xz-utils wget libpq-dev \
+  && wget ${POSTGREST_RELEASES_URL}/${POSTGREST_DOWNLOAD} \
+  && tar xf ${POSTGREST_DOWNLOAD} \
+  && mv postgrest /usr/local/bin/postgrest \
+  && rm -f ${POSTGREST_DOWNLOAD} \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Public env vars, can be passed in as params
+ENV POSTGREST_DB_HOST db
+ENV POSTGREST_DB_PORT 5432
+ENV POSTGREST_DB_USER postgres
+ENV POSTGREST_DB_NAME ${POSTGREST_DB_USER}
+ENV POSTGREST_ANON_USER ${POSTGREST_DB_USER}
+ENV POSTGREST_DB_PASS ${POSTGREST_DB_USER}
+ENV POSTGREST_CONNECTION_POOL_SIZE 10
+ENV POSTGREST_JWT_SECRET secret
+ENV POSTGREST_SCHEMA public
```

Comment:
The port can't be made configurable because the configuration happens at container run-time, after the image is built, while the port exposing happens at image build time. It's not a big deal anyway, because container ports are internal to the container - when running them their exposed ports can be re-mapped by Docker.